quickstart/docker/image/run-router.sh

```diff
@@ -32,6 +36,9 @@ if [[ "${_ZITI_ROUTER_NAME}" != "" ]]; then
 fi
 
 _CONFIG_PATH="${ZITI_HOME}/${ZITI_ROUTER_NAME}.yaml"
+# Set an error trap to move _CONFIG_PATH when it's unsafe to assume enrollment succeeded
+trap '[[ -f "${_CONFIG_PATH}" ]] && mv "${_CONFIG_PATH}" "${_CONFIG_PATH}.err"' ERR
```

Comment:
make the script more resilient against bad state problems resulting from an incomplete run